app/controllers/admin_public_body_categories_controller.rb

```diff
@@ -60,7 +83,17 @@ def update
     def create
         I18n.with_locale(I18n.default_locale) do
             @category = PublicBodyCategory.new(params[:public_body_category])
+            # Build a translation for the default locale as it isn't available
```

Comment:
I think we can just call `@category.translation` here which seems to add the default locale in to the translations array:

```ruby
> category = FactoryGirl.build(:public_body_category)
=> #<PublicBodyCategory id: nil, title: "Example Public Body Category 6", category_tag: "example_tag_6", description: "Example Public body Description 6">
> category.translations
=> []
> category.translation
=> #<PublicBodyCategory::Translation id: nil, public_body_category_id: nil, locale: "en", description: nil, title: nil, created_at: nil, updated_at: nil>
> category.translations
=> [#<PublicBodyCategory::Translation id: nil, public_body_category_id: nil, locale: "en", description: nil, title: nil, created_at: nil, updated_at: nil>]
```